dotcom-rendering/src/web/components/ContainerLayout.tsx

```diff
@@ -162,15 +176,29 @@ export const ContainerLayout = ({
 					stretchRight={stretchRight}
 					format={format}
 				>
-					<Hide when="above" breakpoint="leftCol">
-						<ContainerTitle
-							title={title}
-							fontColour={fontColour || overrides?.text.container}
-							description={description}
-							url={url}
-						/>
-					</Hide>
-					{children}
+					<div css={headlineContainerStyles}>
```

Comment:
I think this is fundamentally good and can be left as is if it's working but we might need to think about what happens when there is more content in the left column. That will push the container down here but we might want to force it closed somehow 🤔 

Anyhoo, unless you have any thoughts or ideas you want to try this can be one for a later PR